lnc/lnc.go

```diff
@@ -281,8 +281,12 @@ func (n *NodeConn) newConn(session *Session, opts ...grpc.DialOption) (*conn,
 	}
 	dialOpts = append(dialOpts, opts...)
 
+	ctxt, cancelT := context.WithTimeout(
+		context.Background(), DefaultConnectionTimetout,
```

Comment:
Since we already have `ctxc` from above, shouldn't we use that as the parent context?